### ISSUE TITLE
Dock Appears in workspaces

### ISSUE THREAD

**Issue by bhillard** [pop-os/gnome-shell-extension-pop-cosmic#46]
The application dock appears when I open  workspaces. It feels unintentional as it covers one of my workspaces.

![Screenshot from 2021-05-23 18-43-21](https://user-images.githubusercontent.com/22432088/119280383-ccf6b000-bbf6-11eb-93fa-9387bce13766.png)

**Comment by blakeridgway** [comment]
I just tested putting the dock to the left hand side, I can not replicate this.

I attempted both hide automatically and always showing and all the default sizes for the bar. 

![image](https://user-images.githubusercontent.com/11250236/119284061-6a0c1580-bc04-11eb-98b6-318d40538700.png)

Having an indication of what your version of pop-cosmic is w/ screen resolution and dock setup may help as well.

![image](https://user-images.githubusercontent.com/11250236/119284282-f1f21f80-bc04-11eb-8636-8092897debf0.png)
